README.md

```diff
@@ -420,6 +433,26 @@ Style: 0 / 0
 Total: 100 / 100
 ```
 
+We can target a specific submission ID through `--target-submission`.
+Use the `autograder.cli.courses.assignments.submissions.fetch.user.history` command to see a student's previous submission IDs and scores.
+Extending the previous example with a target submission ID of `123456789`:
+```sh
+python3 -m autograder.cli.courses.assignments.submissions.proxy.resubmit --proxy-email student@test.edulinq.org --target-submission 123456789
+```
+
+The output may look like (if the target submission was not complete):
+```
+Autograder transcript for assignment: HO0.
+Grading started at 2025-03-04 12:00 and ended at 2025-03-04 12:00.
+Task 1.A (my_function): 40 / 40
+Task 2.A (test_my_function_value): 30 / 30
+Task 2.B (TestMyFunction): 0 / 30
+Style: 0 / 0
+   Style is clean!
+
+Total: 70 / 100
+```
```

Comment:
Like the last section, I'm not sure if the output of the second command gains anything here.